core/gui/package.json

```diff
@@ -131,7 +135,7 @@
     "karma-jasmine": "4.0.1",
     "karma-jasmine-html-reporter": "1.7.0",
     "nodecat": "2.0.0",
-    "nx": "18.1.3",
+    "nx": "18.2.0",
```

Comment:
why are you changing nx version here?

Reply:
Xinyuan told me his branch passed all the checks before, so he think the issue is caused by nx, so we upgrade the version